### DISCUSSION TITLE
PDF titles not showing

### DISCUSSION THREAD

**Discussion by kanotson** [thegetty/quire#731] [Feedback]
Hi,

I've tried Quire on a Mac with more success than Windows. Now it builds the project OK and the PDF has images. But the titles in the PDF are not visible (it shows only the second lines of the longest subtitles). Is it a bug, or a problem with my content?

**Comment by geealbers** [comment]
Hi @kanotson, that's not something I think we've seen before. Could you share a sample of the PDF and the corresponding Markdown file (or just the block of YAML at the top) for where you're seeing this?

**Reply by kanotson** [reply]
Yes, here you can download the [PDF and two markdown files](https://pradocloud.museodelprado.es/owncloud/index.php/s/zQVSHq092KDBuHr): digitalizacion1.md and digitalizacion2.md. The first chapter appears properly in the PDF, but the second one is not showing the title (page 7 of the PDF).

**Reply by geealbers** [reply]
Those samples were helpful, thank you. I believe the issue is being caused by some CSS styling issues which we have recently fixed. I would recommend you update your project following the instructions in our documentation to get the latest changes which include a number of improvements to the PDF output: https://quire.getty.edu/docs-v1/install-uninstall/#update-an-existing-project.

Let us know if this works for you, or if you run into any other issues.

**Reply by kanotson** [reply]
I've tried to update the project, but with the same results. 
Do you have an estimated date for the release of a stable version? Or "is done when is done"?

**Reply by geealbers** [reply]
@kanotson I'm sorry to hear the update didn't fix the problem. 

One other option you could try is using [PrinceXML](https://www.princexml.com/) to generate the PDF. You can download and use if for free for testing and non-commercial use, or a desktop license costs $500 otherwise. Once installed, you can generate the PDF with Prince with a flag on the regular command: `quire pdf --lib prince`.

Also, I'd be happy to look at the issue again, but would need your full project files. You could sent them to us, but without the `_epub`, `_site`, `public`, and `node_modules` directories if they exist. (Note that you can reinstall `node_modules` anytime by running `npm install`, and the other folders are regenerated by Quire as they are needed. Deleting them before sending to us just saves a lot of space.) And feel free to send them via a download link or WeTransfer to quire@getty.edu rather than posting them here if you'd like.

As for a stable release date, I'm afraid we don't have one. We are working on bug fixes and new features regularly and still have a ways to go before we'll feel we've got the majority of them done.

**Comment by Erin-Cecele** [comment]
Hi @kanotson, was this issue ever resolved? Is there anything else we can assist with?
